js/start.js

```diff
@@ -85,6 +85,8 @@ if (process.platform === 'darwin') {
 
 $html.addClass(platformClass);
 
+app.bitcoinUnit = localStorage.getItem('BitcoinUnit') || 'BTC';
```

Comment:
Rather than the BitcoinUnit having to be retrieved and set in two places (app.js and localStorage), can we please centralize that in app.js, e.g:

```
// app.js
App.prototype.getBitcoinUnit = function() {
  return localStorage.getItem('bitcoinUnit') || 'BTC';
}

App.prototype.setBitcoinUnit = function(unit) {
  localStorage.setItem('bitcoinUnit', unit);
}
```

With this in place, you can eliminate the above line from start.js and any areas of code that need the bitcoinUnit, can retrieve it from our App instance.